src/pages/Defi/components/EligibleCarousel.tsx

```diff
@@ -42,7 +42,12 @@ export const EligibleCarousel: React.FC<EligibleCarouselProps> = props => {
   const filteredEligibleOpportunities = useMemo(() => {
     // opportunities with 1% APY or more
     const filteredEligibleOpportunities = eligibleOpportunities
-      .filter(o => bnOrZero(o.tvl).gt(50000) && bnOrZero(o.apy).gte(0.01))
+      .filter(
+        o =>
+          bnOrZero(o.tvl).gt(50000) &&
+          bnOrZero(o.apy).gte(0.01) &&
+          o.provider !== DefiProvider.ThorchainSavers,
```

Comment:
ditto remark re: flag/mimir vs. hardcoded magic

Reply:
Done in [59f35c1](https://github.com/shapeshift/web/pull/8681/commits/59f35c1b7b628ff50975e1d958516465cee23df4)